server/src/main/java/org/apache/druid/server/coordinator/CuratorLoadQueuePeon.java

```diff
@@ -108,6 +104,18 @@ private static void executeCallbacks(List<LoadPeonCallback> callbacks)
     this.callBackExecutor = callbackExecutor;
     this.processingExecutor = processingExecutor;
     this.config = config;
+    this.monitorNodeRemovedExecutor =
```

Comment:
https://github.com/code-review-checklists/java-concurrency#threading-flow-model - missing in this class